src/steps/email-count-equals.ts

```diff
@@ -56,23 +78,47 @@ export class EmailCountEqualsStep extends BaseStep implements StepInterface {
         ]);
       }
 
+      const records = this.createRecords(inbox.items);
+
       // tslint:disable-next-line:triple-equals
       if (inbox.items.length == stepData.count) {
-        return this.pass('Found %d emails, as expected', [
-          inbox.items.length,
-        ]);
+        return this.pass(
+          'Found %d emails, as expected',
+          [inbox.items.length],
+          [records],
+        );
       } else {
-        return this.fail('Expected there to be %d emails, but there were actually %d. Their subjects are:\n\n%s', [
-          stepData.count,
-          inbox.items.length,
-          inbox.items.map(f => f['message']['headers']['subject']).join(EOL),
-        ]);
+        return this.fail(
+          'Expected there to be %d emails, but there were actually %d. Their subjects are:\n\n%s',
```

Comment:
- [ ] We can get rid of this `Their subjects are` portion of this, since we're printing the table now.